README.md

```diff
@@ -64,32 +64,56 @@ you need to add PYTHONUNBUFFERED=1;PYDEVD_USE_FRAME_EVAL=NO to your Run/Debug en
     - **files:**
       [TheBloke/Mistral-7B-Instruct-v0.2-GGUF/tree/main](https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.2-GGUF/tree/main)
 
-## Run with docker (** NOT IMPLEMENTED **)
+## Run with docker
 
-- **CPU Mode**
+- **CPU Mode**  (** NOT IMPLEMENTED **)
     - CPU - slow but reliable
         - ```
-            docker run --name alexandria-project alexandria-project -v path/to/model:/usr/model --env-file https://raw.githubusercontent.com/alexandriaproject-io/llm-inference/main/env-samples/.env.cpu.example 
+            docker build -f docker/Dockerfile.cuda12 -t alexandria_project .
+
+            docker run -p 5050:5050 --env-file ./env-samples/.env.auto.example \
+        -v $(pwd)/TinyLlama-1.1B-Chat-v1.0:/usr/src/app/app/TinyLlama-1.1B-Chat-v1.0 \
+        -e MODEL_PATH=TinyLlama/TinyLlama-1.1B-Chat-v1.0 \
+        --name alexandria_projecdt_instance \
```

Comment:
Please simplify this example to one liner like we talked about

Reply:
fixed